commands/Core/dblog.b.inc

```diff
@@ -49,7 +44,8 @@ function dblog_show_many($options) {
     $options['count'] = 10;
   }    
   $query = db_select('watchdog', 'wd')->extend('PagerDefault')
-    ->fields('wd');
+    ->fields('wd')
+    ->orderBy('wid', DESC);
```

Comment:
Use of undefined constant DESC - assumed 'DESC'
	/usr/local/bin/b/commands/Core/dblog.b.inc:48

;)

Reply:
Thanks @indigoxela, fixed!